Variables.CMakeFile

```diff
@@ -72,6 +73,7 @@ dir2macro(OIO__SERVER_POOL_MAX_TCP)
 dir2macro(OIO__SERVER_POOL_MAX_UDP)
 dir2macro(OIO__SERVER_POOL_MAX_UNUSED)
 dir2macro(OIO__SERVER_QUEUE_MAX_DELAY)
+dir2macro(OIO__SERVER_QUEUE_MAX_DELAY)
```

Comment:
This file is auto-generated, the error is a consequence of your previous message.